addons/udes_security/controllers/redirect.py

```diff
@@ -51,6 +53,9 @@ class Home(main.Home):
     @no_absolute_redirect
     def web_client(self, *args, **kwargs):
         """Prevent absolute redirects on /web"""
```

Comment:
This docstring could do with an update

Reply:
This method has two purposes now, not just one! It still prevents absolute redirects